src/codegen/wasm.zig

```diff
@@ -81,7 +114,10 @@ fn genInst(buf: *ArrayList(u8), decl: *Decl, inst: *Inst) !void {
         .dbg_stmt => {},
         .ret => genRet(buf, decl, inst.castTag(.ret).?),
         .retvoid => {},
-        else => error.TODOImplementMoreWasmCodegen,
+        .alloc => genAlloc(buf, decl, inst.castTag(.alloc).?),
+        .store => genStore(buf, decl, inst.castTag(.store).?),
+        .load => genLoad(buf, decl, inst.castTag(.load).?),
+        else => return error.TODOImplementMoreWasmCodegen,
```

Comment:
This return isn't necessary.
```suggestion
        else => error.TODOImplementMoreWasmCodegen,
```